### ISSUE TITLE
Python wrapper

### ISSUE THREAD

**Issue by AzHicham** [Tudyx/ai-dataloader#49]
Hello,

I was wondering if we can create a python wrapper to this library using pyO3/maturin and ship it on Pypi.
This way python user could also benefit from this crate ie no more process fork but real multithreading.

**Comment by Tudyx** [comment]
Yes, that's definitely something that I had in mind when I started the project. I will be open to maintain it if someone wants to submit a PR
